ui/app/components/modals/deposit-ether-modal.js

```diff
@@ -29,6 +31,9 @@ function mapDispatchToProps (dispatch) {
     toCoinbase: (address) => {
       dispatch(actions.buyEth({ network: '1', address, amount: 0 }))
     },
+    toCoinSwitch: (address) => {
+      dispatch(actions.buyEth({ network: '5', address, amount: 0 }))
```

Comment:
`network` indicates the ethereum `networkId` ( mainnet vs testnets ). I suggest adding a new field like `service: 'coinswitch'` and using that to generate the url in `buy-eth-url.js`

Reply:
@kumavis I have made the changes. Please have a look.